package.json

```diff
@@ -5,12 +5,17 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "test": "jest --watch",
-    "test/prod": "jest",
-    "lint": "eslint ./"
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "lint": "eslint ./",
+    "format:verify": "prettier --list-different '{./components,./pages,./__tests__,./docs}/**/*.js'",
+    "format": "prettier --write '{./components,./pages,./__tests__,./docs}/**/*.js'"
   },
   "dependencies": {
```

Comment:
Would be good to split this into dependencies / dev dependencies sooner rather than later - would allow us to grok if our package is gonna get huge.

We should also be targeting specific versions of libaries (remove the ^ basically) so that unexpected upgradates don't happen

Reply:
Is this still best practice even with the yarn.lock?

Reply:
Fair point, probably not relevant then. For OCD reasons would be nice to have them either with or without carets 😛 

Only edge case I can think of for removing carets is if someone comes along and uses npm instead of yarn. Looking at other projects like jest / yarn themselves they don't seem to care about that though.

Reply:
Will do this

Reply:
In a separate branch